dd-sdk-android/src/main/json/rum/_action-child-schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "rum/_action-child-schema.json",
+  "title": "ActionChildProperties",
+  "type": "object",
+  "description": "Schema of all properties of events that can have parent actions",
+  "properties": {
+    "action": {
+      "type": "object",
+      "description": "Action properties",
+      "required": ["id"],
+      "properties": {
+        "id": {
+          "description": "UUID of the action",
+          "readOnly": true,
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$"
+            },
+            {
+              "type": "array",
```

Comment:
are we using this event ? I don't remember to have something like this yet in the SDK.

Reply:
This schema is shared for all action children (i.e. Resources, Errors and Long Tasks). we used to only have an `action.id: String`, but now it can be either a String, or an array of Strings (for Frustration Signals)